developer/web-api/metadata.md

```diff
@@ -1411,6 +1411,37 @@ Table: Organisation unit parameters
 | includeAncestors | false &#124; true | Include all parents of the specified organisation unit. |
 | level | integer | Include children of the specified organisation unit at the given level of the sub-hierarchy (relative to the organisation unit where the immediate level below is level 1). |
 
+### Get organisation units by categoryOption  { #webapi_organisation_units_by_categoryOptions }
+
+Purpose-built endpoint to retrieve associations between CategoryOptions and Organisation Units. 
+This endpoint is the preferred way to retrieve program org unit associations.
```

Comment:
Try to be consistent when referring to the different entities:
CategoryOption / Organisation Units -> OrganisationUnits
program org unit associations -> retrieve associations between Program and OrganisationUnits.